src/main/scala/zio/kafka/admin/AdminClient.scala

```diff
@@ -804,6 +860,39 @@ object AdminClient extends Accessible[AdminClient] {
     lazy val asJava: JDeleteConsumerGroupsOptions = new JDeleteConsumerGroupsOptions().timeoutMs(timeout.toMillis.toInt)
   }
 
+  final case class AlterConfigsOptions(validateOnly: Boolean = false, timeout: Option[Duration] = None) {
+    lazy val asJava: JAlterConfigsOptions = {
+      val jOpts = new JAlterConfigsOptions().validateOnly(validateOnly)
+      timeout.fold(jOpts)(timeout => jOpts.timeoutMs(timeout.toMillis.toInt))
+    }
+  }
+
+  final case class AlterConfigOp(configEntry: ConfigEntry, opType: AlterConfigOpType) {
+    lazy val asJava: JAlterConfigOp = new JAlterConfigOp(configEntry, opType.asJava)
+  }
+
+  trait AlterConfigOpType {
+    def asJava: JAlterConfigOp.OpType
+  }
+
+  object AlterConfigOpType {
+    case object Set extends AlterConfigOpType {
+      lazy val asJava = JAlterConfigOp.OpType.SET
```

Comment:
```suggestion
      val asJava = JAlterConfigOp.OpType.SET
```

Reply:
It is how this val is implemented for every other case class/object. Even if I agree, I don't want to diverge from the original repo :)